app/Http/Controllers/RestaurantsController.php

```diff
@@ -43,54 +37,56 @@ public function create()
-        //Get just filename
-        $menuname=pathinfo($menunameWithExt,PATHINFO_FILENAME);
-        //Get just ext
-        $extension=$request->file('menu_image')->getClientOriginalExtension();
-        //Filename to store
-        $menuNameToStore=$filename.'_'.time().'_'.$extension;
-        //Upload image
-        $path=$request->file('menu_image')->storeAs('public/cover_images',$menuNameToStore);   
-    }
-    else 
-    $menuNameToStore='noImage.jpg';
+    // //handle file upload
+    // if($request->hasFile('cover_image')){
+    //     // Get filename with the extension
+    //     $filenameWithExt=$request->file('cover_image')->getClientOriginalName();
+    //     //Get just filename
+    //     $filename=pathinfo($filenameWithExt,PATHINFO_FILENAME);
+    //     //Get just ext
+    //     $extension=$request->file('cover_image')->getClientOriginalExtension();
+    //     //Filename to store
+    //     $fileNameToStore=$filename.'_'.time().'_'.$extension;
+    //     //Upload image
+    //     $path=$request->file('cover_image')->storeAs('public/cover_images',$fileNameToStore);
+    // }
+    // {
+    //  //handle file upload
+    // if($request->hasFile('menu_image')){
+    //     // Get filename with the extension
+    //     $menunameWithExt=$request->file('menu_image')->getClientOriginalName();
+    //     //Get just filename
+    //     $menuname=pathinfo($menunameWithExt,PATHINFO_FILENAME);
+    //     //Get just ext
+    //     $extension=$request->file('menu_image')->getClientOriginalExtension();
+    //     //Filename to store
+    //     $menuNameToStore=$filename.'_'.time().'_'.$extension;
+    //     //Upload image
+    //     $path=$request->file('menu_image')->storeAs('public/cover_images',$menuNameToStore);   
+    // }
+    // else 
+    // $menuNameToStore='noImage.jpg';
 
     {
         //Create Restaurant
         $restaurant=new Restaurant;
         $restaurant->name= $request->input('name');
-        $restaurant->menu=$request->input('menu');
-        $restaurant->hours=$request->input('hours');
-        $restaurant->contact=$request->input('contact');
-        $restaurant->payment=$request->input('payment');
-        $restaurant->description=$request->input('description');
-        $restaurant->cover_image=$fileNameToStore;
-        $restaurant->menu_image=$menuNameToStore;
+        $restaurant->opening_time=$request->input('opening_time');
+        $restaurant->physical_address=$request->input('physical_address');
+        $restaurant->closing_time=$request->input('closing_time');
+        $restaurant->days_of_operation=$request->input('days_of_operation');
+        $restaurant->website=$request->input('website');
+        $restaurant->email=$request->input('email');
+        $restaurant->phone=$request->input('phone');
+        // $restaurant->cover_image=$fileNameToStore;
+        // $restaurant->menu_image=$menuNameToStore;
         $restaurant->save();
  
         return redirect("profile/");   
```

Comment:
Could you make this to redirect to **restaurants/** instead of the users profile. Right now if you save a new restaurant it goes to the users profile instead of the main page containing the list of restaurants.

Reply:
fixed it